rust/chains/hyperlane-ethereum/src/interchain_security_module.rs

```diff
@@ -96,9 +97,14 @@ where
     M: Middleware + 'static,
 {
     #[instrument(err, ret)]
-    async fn module_type(&self) -> ChainResult<u8> {
-        let module_type = self.contract.module_type().call().await?;
-        Ok(module_type)
+    async fn module_type(&self) -> ChainResult<ModuleType> {
+        let module = self.contract.module_type().call().await?;
+        if let Some(module_type) = ModuleType::from_u8(module) {
+            Ok(module_type)
+        } else {
+            warn!("Unknown module type: {}", module);
```

Comment:
Formatting content into the message string makes it hard to filter in gcp
```suggestion
            warn!(%module, "Unknown module type");
```